salesagility/SuiteCRM
# Two Factor Auth via Code in Email

## Description




Two Factor Auth via Code in Email

## Motivation and Context


ref: SCRM-742

## How To Test This

- set email template for two factor authentication email on admin / password settings page
- set two factor authentication checkbox on user profile page
- logout
- login
- check your emails
- use the given code to finalise login process

## Types of changes


### Final checklist
